util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/AttributesUtil.java

```diff
@@ -151,11 +151,7 @@ public static String attributesToQidoPath(Attributes attrs, String... includeFie
 
       for (String value : values) {
         String encodedValue;
-        try {
```

Comment:
Why was this change made?

Reply:
URLEncoder.encode(value, "UTF-8") is older syntax that needs to be wrapped into try-catch,  URLEncoder.encode(value, StandardCharsets.UTF_8) doesn't need try-catch and so is cleaner.

Not really necessary to current feature, just minor refactoring.

Reply:
Got it thanks.